webpack.config.js

```diff
@@ -38,6 +51,16 @@ if (production) {
     test: /\.scss/,
     use: extractSass.extract({
       use: [
+        {
+          loader: 'string-replace-loader',
```

Comment:
I tried creating a custom SASS function, but for some reason it was being ignored by SASS. I spent over an hour on this, trying all sorts of things, including upgrading `sass-loader`.